### PR TITLE
Portfolio

### PR DESCRIPTION
**Pull request by JansueT**
### This pull request about 
- Add index.html 
- Add style.css
- Add linters for checking errors
- Figma

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    </header>
+    <section class="headline">
+        <h1 class="headline_title"><span class="sub-headline_title_1">Hey there. I’m Mariane</span><span class="sub-headline_title">I'm a software developer</span></h1>
+        <p class="headline_content"> I can help you build a product ,
+            feature or website Look through
+             some of my 
+            work and experience!
+            If you like what you see and have
+             a project you need coded,
+            don’t hestiate to contact me. </p>
```
Comment (ha-manel) [inline index.html:30]: - [ ] Great job on using the right HTML tags 👏 Kindly follow the Figma design by adding the appropriate `font-family` to all the text in the page (including "Welcome"). 

## Your design:
image

## Figma design:
image

Comment (ha-manel) [inline index.html:30]: - [ ] Kindly increase the `font-size` of the paragraph to match the text on Figma 👍 

## Your design:
image

## Figma design:
image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+                        <img class="social_icon" src="image/twitter.svg" alt="twitter">
+                    </a>
+                </li>
+                <li class="social_links">
+                <li class="social_link">
+                    <a href="#">
+                        <img class="social_icon" src="image/linkedin.svg" alt="linkedin">
+                    </a>
+                </li>
+                </li>
```
Comment (ha-manel) [inline index.html:44]: - [ ] Kindly remove this `li` that is wrapping the 2nd `li` 👍

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi @JansueT👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  



## Highlights:  
* All tests passed ✔️
* The website looks great 💯 

## Required Changes:
- [x] Kindly remove the placeholder text in the "Getting Started" section of the README file and fill it out with information on how to setup your project locally (I couldn't put this in inline comments because it's not in the changed files so I'm mentioning it here 👍).
- [ ] From now on kindly write the commit messages as capitalized sentences in the imperative form (ex: "Add linters" and not "added linters"...) 

_Kindly check the comments below_

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @JansueT, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

-  No linter errors. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 
-   Good README file. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**